core/src/main/java/com/linecorp/armeria/server/ServerHttp1ObjectEncoder.java

```diff
@@ -46,6 +46,8 @@
 
     private boolean sentConnectionCloseHeader;
 
+    private int lastResponseHeadersId;
```

Comment:
nit: Use `-1` for the initial value?

Reply:
Hm, I used the same value here https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/server/Http1RequestDecoder.java#L91

Reply:
OK then, it makes sense.